### DISCUSSION TITLE
Tooltip is not working with Dialog on MUI

### DISCUSSION THREAD

**Discussion by Suyama-Daichi** [airbnb/visx#1634] [General]
Thank u nice package.

I use `@visix/xychart` in Dialog with MUI but it can't shown `Tooltip`.

Do u have any solution ?

example:
<s>https://codesandbox.io/s/linechart-with-dialog-4wu7ik</s>
-> solved!

Thank u.

**Comment by SGEMKE** [comment]
Hey,

the tooltip is correctly generated, you just can't see it because the mui dialog is forced to the very front. You can check this by making the dialog transparent.
`      <Dialog open={open} fullWidth={true}   PaperProps={{
    style: {
      backgroundColor: 'transparent'
    },
  }}>`

if you sort out the z-index of your elements, everything should work.

**Comment by Suyama-Daichi** [comment]
I handled z-index of Modal.
```
    const theme = createTheme({
        zIndex: { modal: 0.3, drawer: 0.1, appBar: 0.2 },
    })
```

![スクリーンショット 2023-01-23 12 30 40](https://user-images.githubusercontent.com/35884504/213961357-a9fcc20b-290b-4c6a-9d0c-9f8ff661479f.png)

It correctly work now!
